packages/hashes.csv

```diff
@@ -19,7 +19,7 @@ fetchai/connections/p2p_client,QmdTkNaZaGYQ6UFWcNvyNf9qW6jFEQr8kd5fHcPk57gfsj
 fetchai/connections/scaffold,QmT8vqahQ8K7KB98xHuxVRAVkrcky9xaVFXMcsBNtbPfM4
 fetchai/connections/stub,QmToaVJa3Jktq9qdgJ2TjnHHwtT1Cf57oNjN4qHnCvTeD5
 fetchai/connections/tcp,QmS2rmJ9PX2ukS6TqLpUWgRKpt6GTnoHQYnY64XeJb6sDK
-fetchai/contracts/erc1155,QmdvrdhUiVh1E6SSvmfKc1CmWf1tF3Aqt1eNQsdoSFJDxC

```

Comment:
I don't understand why there are differences in hashes, that should not be the case.